plugins/blame.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: blame
+spec:
+  version: v0.0.5
+  homepage: https://github.com/knight42/kubectl-blame
+  shortDescription: Show the manager of each field of given resource with information from managed fields.
+  description: |
+    Annotate each line in the given resource's YAML with information from the managedFields to show who last modified the field.
+
+    Examples:
+
+    # Blame pod 'foo' in default namespace
+    kubectl blame pods foo
+
+    # Blame deployment 'bar' in 'ns1' namespace
+    kubectl blame -n ns1 deploy bar
```

Comment:
typically it's better if you repo or --help message contains these examples.
You can remove these.